pages/settings/wallets/[wallet].js

```diff
@@ -67,10 +67,16 @@ export default function WalletSettings () {
     }
   }, [wallet.def])
 
+  const { card: { image, title, subtitle } } = wallet?.def || { card: {} }
```

Comment:
```suggestion
  const { card: { image, title, subtitle } = {} } = wallet?.def
```

Are there wallet defs without cards now?

Reply:
Mhh, no, I just saw us using `wallet?.def` a lot so I wanted to assume `wallet?.def` can actually be `undefined` in some case here so I added `|| { card: {} }`. But your solution looks better and I didn't think much about this line.

Imo, we use optional chaining in a lot of places where it shouldn't be needed but it's a trade-off between failing fast (= crashing) and subtle bugs so I can see the point of using it even when in theory, it shouldn't be needed.

Reply:
Oh right, that would cause a crash. I got in the optional chaining habit back when I was learning react a few years ago. It'd probably be better to do explicit checks or something now.